app/globals.css

```
@@ -375,3 +375,558 @@ h6 {
+  main {
+    flex: 1;
+    overflow-y: auto;
+    -webkit-overflow-scrolling: touch;
+  }
+}
+
+/* Mobile Browser - Force Hide Address Bar WITH SCROLLING */
+@media screen and (max-width: 768px) {
+  html {
+    height: 100vh;
+    height: -webkit-fill-available;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

height is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)